testutil/ibc/testchain.go

```diff
@@ -161,7 +161,7 @@ func SignAndDeliver(
 	chainID string, accNums, accSeqs []uint64, _ bool, blockTime time.Time, nextValHash []byte, priv ...cryptotypes.PrivKey,
 ) (*abci.ResponseFinalizeBlock, error) {
 	tx, err := simtestutil.GenSignedMockTx(
-		rand.New(rand.NewSource(1)),
+		rand.New(rand.NewSource(1)), //nolint:gosec // G304
```

Comment:
G304 doesn't make sense for this line, is that the right thing to put here?

G304 is about a tained filepath being used, but this is just a random number generator line.